app/uk/gov/hmrc/economiccrimelevyenrolment/forms/mappings/LocalDateFormatter.scala

```diff
@@ -27,6 +27,7 @@ import scala.util.{Failure, Success, Try}
 private[mappings] class LocalDateFormatter(
   invalidKey: String,
   requiredKey: String,
+  sanitise: String => String,
```

Comment:
This is appropriate here, as it's a function which could accept any type of sanitise operation. In my comment above, the function you pass in though, in the form provider should be named against what it does.

Reply:
Agreed and corrected.